plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataSessionProperties.java

```diff
@@ -65,6 +66,12 @@ public JdbcMetadataSessionProperties(JdbcMetadataConfig jdbcMetadataConfig, @Max
                         "Enable TopN pushdown",
                         jdbcMetadataConfig.isTopNPushdownEnabled(),
                         false))
+                .add(integerProperty(
+                        INSERT_BATCH_SIZE,
+                        "Insert batch size",
+                        jdbcMetadataConfig.getInsertBatchSize(),
+                        value -> validateInsertBatchSize(value, JdbcMetadataConfig.MAX_ALLOWED_INSERT_BATCH_SIZE),
```

Comment:
nit: prefer using static imports where the short name isn't ambiguous

Reply:
done